src/EnergyPlus/PlantChillers.cc

```diff
@@ -108,47 +108,7 @@ namespace PlantChillers {
     // 1. BLAST Users Manual
 
     // Parameters for use in Chillers
-    Real64 const KJtoJ(1000.0); // convert Kjoules to joules
-
-    // chiller flow modes
-    int const FlowModeNotSet(200);
-    int const ConstantFlow(201);
-    int const NotModulated(202);
-    int const LeavingSetPointModulated(203);
-
-    static std::string const BlankString;
-
-    int NumElectricChillers(0);     // number of Electric chillers specified in input
-    int NumEngineDrivenChillers(0); // number of EngineDriven chillers specified in input
-    int NumGTChillers(0);           // number of GT chillers specified in input
-    int NumConstCOPChillers(0);
-
-    bool GetEngineDrivenInput(true); // then TRUE, calls subroutine to read input file.
-    bool GetElectricInput(true);     // then TRUE, calls subroutine to read input file.
-    bool GetGasTurbineInput(true);   // then TRUE, calls subroutine to read input file.
-    bool GetConstCOPInput(true);
-
-    // Object Data
-    Array1D<ElectricChillerSpecs> ElectricChiller;         // dimension to number of machines
-    Array1D<EngineDrivenChillerSpecs> EngineDrivenChiller; // dimension to number of machines
-    Array1D<GTChillerSpecs> GTChiller;                     // dimension to number of machines
-    Array1D<ConstCOPChillerSpecs> ConstCOPChiller;         // dimension to number of machines
-
-    void clear_state()
-    {
-        NumElectricChillers = 0;
-        NumEngineDrivenChillers = 0;
-        NumGTChillers = 0;
-        NumConstCOPChillers = 0;
-        GetEngineDrivenInput = true;
-        GetElectricInput = true;
-        GetGasTurbineInput = true;
-        GetConstCOPInput = true;
-        ElectricChiller.deallocate();
-        EngineDrivenChiller.deallocate();
-        GTChiller.deallocate();
-        ConstCOPChiller.deallocate();
-    }
+    Real64 constexpr KJtoJ(1000.0); // convert Kjoules to joules
```

Comment:
```constexpr``` for numerical constants. There's probably something similar for the string constants.